tests/e2e-kind.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
-
+CI="${CI:-false}"
```

Comment:
who is setting the CI variable to `true` in the actual CI environment? can we add a doc reference?

Reply:
All CI environments set the variable.